.github/scripts/ep_hooks.py

```
@@ -0,0 +1,267 @@
+        verdict_path = Path(work_dir) / "verdict.json"
+        if not verdict_path.exists():
+            raise FileNotFoundError(f"verdict.json not found in {work_dir}")
+        with open(verdict_path) as f:
+            verdict = json.load(f)
+        if "scores" not in verdict or "verdict" not in verdict:
+            raise ValueError("verdict.json missing required fields")
+        return verdict
+
+    # ── Post-gate ──
+
+    def validate_scores(self, ticket_key, ticket=None, mode=None,
```

Ruff (0.15.20)

[warning] Unused method argument: `ticket_key`

(ARG002)